utils/middleware/error-handling-middleware.ts

```diff
@@ -0,0 +1,41 @@
+import middy from "@middy/core";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { serializeError } from "serialize-error";
+import { ValidationErrorResponse } from "../response";
+import { INTERNAL_SERVER_ERROR, REQUEST_BODY_INVALID } from "../errors";
+
+export const errorHandlingMiddleware = (): middy.MiddlewareObj<APIGatewayProxyEvent, APIGatewayProxyResult> => {
+  const onError: middy.MiddlewareFn<APIGatewayProxyEvent, APIGatewayProxyResult> = async (
+    request
+  ): Promise<ValidationErrorResponse | void> => {
+    const error = serializeError(request.error)!;
+    const errorMessage = error?.message as string;
```

Comment:
`foo?.attr as` <something that doesn't include undefined> is inherently unsafe, right?

Reply:
Yeah, that makes sense. Replaced with simply `error.message`.